### PR TITLE
add apidoc publish flag

### PR DESCRIPTION
**Pull request by ebarboni**
add this flag to handle apidoc generation better

### CODE REVIEW

File: meta/netbeansrelease.json [hunk 1]
```diff
@@ -574,7 +589,7 @@
     "master": {
         "position": "50000",
         "ant": "ant_latest",
-        "jdk": "jdk_1.8_latest",
+        "jdk": "jdk_1.11_latest",
```
Comment (neilcsmith-net) [inline meta/netbeansrelease.json:592]: Not `jdk_11_latest`?

**Review by neilcsmith-net — APPROVED** [review]
Looks good, apart from possibly the inconsistency with JDK 11 naming (change on master is good, just looking at builds on Travis and Github for this).

I would still like to see VSNetBeans specify and use an alternative json - `vsnetbeansrelease.json` ?

File: meta/netbeansrelease.json (hunk 1 above)
Comment (ebarboni) [reply meta/netbeansrelease.json]: yep your right.

Fixing release date for 12.6 in the same time..

for the vsnetbeansrelease.json we have some common item? (there is release on netbeans release and another before ? )

Comment (neilcsmith-net) [reply meta/netbeansrelease.json]: Yes, as far as I know. Have suggested someone involved joins IDE release team so we can coordinate.

Separate file would stop those polluting our release info, but require a bit of duplication.  There are a few other things, like the previous releases in the IDE includes the VSNetBeans releases.  Leaves us free to rely on only information that's relevant for the IDE in here.

Comment (ebarboni) [reply meta/netbeansrelease.json]: yep sounds good. merging this for now

**Pull request merged**